### ISSUE TITLE
Show line numbers to the left of code extract

### ISSUE THREAD

**Issue by yawaramin** [Duhemm/sbt-errors-summary#26]
Currently we show line numbers at the end of the file name:

```
[1] path/to/File.scala:21:
    not found: value foo
      foo()
      ^
```

This makes it difficult to quickly find the line number because we have to scan across the entire file name looking for the end of the line. It's a small problem for simple path hierarchies but gets bigger as projects get more complex package hierarchies.

I propose copying Elm and Rust and putting the line number immediately to the left of the code extract:

```
[E1] path/to/File.scala:
     not found: value foo
     L21:  Bar.foo()
               ^
```

From the familiarity perspective as well, people are more used to seeing line numbers to the left of their code.

**Comment by Duhemm** [comment]
That's a pretty good idea, and it would certainly fix #12. PR welcome!

**Comment by yawaramin** [comment]
Cool, I'll give it a shot :-)
